src/core/libraries/save_data/save_memory.cpp

```diff
@@ -196,9 +199,10 @@ void ReadMemory(u32 slot_id, void* buf, size_t buf_size, int64_t offset) {
     auto& data = g_attached_slots[slot_id];
     auto& memory = data.memory_cache;
     if (memory.empty()) { // Load file
+        memory.resize(data.memory_cache_size);
+        memset(memory.data(), 0, data.memory_cache_size);
```

Comment:
When resizing, the vector already sets the elements as default-initialization, so don't need to memset again